lib/public/css/nav.scss

```diff
@@ -1,50 +1,117 @@
 .nav {
-    padding: 0 3em;
-    display: flex;
+  background-color: $dark-orange;
+  border-bottom: 2px solid $tan;
+  display: flex;
 
-    .nav__logo-wrapper {
-        display: flex;
-        align-items: center;
-        flex: auto 0;
+  @media #{$xsmall-only} {
+    display: block;
+  }
+
+  &__logo {
+    height: 4em;
+    margin: 2em;
+
+    &__wrapper {
+      align-items: center;
+      display: flex;
+      flex: auto 0;
+
+      @media #{$xsmall-only} {
+        justify-content: center;
+      }
 
-        .nav__logo {
-            height: 4em;
-            margin: 1.5em 0;
-        }
     }
 
-    .nav__item-wrapper {
-        display: flex;
-        align-items: center;
-        flex: auto;
+  }
 
-        > div {
-            display: flex;
-            flex-direction: row;
-            flex-wrap: wrap;
-            justify-content: flex-end;
-            flex: auto;
-        }
+  &__icon {
+    margin-left: 1rem;
+    position: relative;
+    top: -10px;
+    width: 2rem;
 
-        a.nav__item {
-            display: inline-block;
-            padding: .5rem;
+    @media #{$small-up} {
+      padding: .25rem 1rem 0 0;
+      top: 0;
+    }
 
-            .nav__item-dropdown {
```

Comment:
Similarly, I was trying to make `item-dropdown` the BEM element (and `nav` the block).  If this is wrong, what is a proper way to write that?

Reply:
Since it's an element, it'd be `.nav__item__dropdown` you can stack elements. or you can modify the item to become a dropdown. Usually modifiers are less element based and more styles, or tweaks to a element that slightly modifies it. (at least my understanding)

Reply:
Ah, gotchya– cool!

Reply:
Interesting! In my experience, I try not to stack elements. I looked at the documentation out of curiosity, and the official response to this question is, "[According to BEM method, block structure should be flattened](http://getbem.com/faq/#css-nested-elements)," so instead of `.nav__item__dropdown`, you could just do `.nav__item` and `.nav__dropdown`.

Reply:
Also, I hope I'm not totally out of place to comment on this—I can't seem to get enough BRS 😄

Reply:
great point man! Sorry just seeing this now, I agree!

Reply:
Thumbs up! This thing is going to look great :)